ts/src/bybit.ts

```diff
@@ -7583,13 +7584,28 @@ export default class bybit extends Exchange {
         const request = {
             'mode': mode,
         };
-        if (symbol === undefined) {
-            request['coin'] = 'USDT';
-        } else {
-            const market = this.market (symbol);
+        const enableUnified = await this.isUnifiedEnabled ();
+        let method = undefined;
+        if (enableUnified[1]) {
+            this.checkRequiredSymbol ('setPositionMode', symbol);
+            market = this.market (symbol);
+            if (!market['inverse']) {
+                throw new NotSupported (this.id + ' setPositionMode () only support inverse market for unified trading account');
```

Comment:
actually I will remove these check, I managed to change the positionMode of a linear swap using a unified account

Reply:
```
n bybit setPositionMode true "BTC/USDT:USDT" --sandbox
Debugger attached.
2023-05-21T10:04:07.315Z
Node.js: v18.14.0
CCXT v3.1.2
bybit.setPositionMode (true, BTC/USDT:USDT)
2023-05-21T10:04:09.719Z iteration 0 passed in 591 ms

{
  retCode: '0',
  retMsg: 'OK',
  result: {},
  retExtInfo: {},
  time: '1684663449717'
}
2023-05-21T10:04:09.719Z iteration 1 passed in 591 ms
```